### DISCUSSION TITLE
How to set CS:GO server name?

### DISCUSSION THREAD

**Discussion by fstltna** [GameServerManagers/LinuxGSM#3864] [Q&A] [answered]
No-matter what I do the server name shows up as LinuxGSM, not my desired name. Where do I set it?

**Comment by ElZeRo69** [comment, marked as answer]
you must edit "../serverfiles/csgo/cfg/csgoserver.cfg"

// .................................. Basic ................................. //
// Hostname - Name of the server.
hostname "fstlna' super csgoserver"
